src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,2 @@
+spring.sql.init.mode=always
```

Comment:
Jeżeli korzystamy z liquibase, to schemat i dane testowe powinny być zarządzanie przez niego zarówno dla profilu deweloperskiego, jak i produkcyjnego, więc ta własność jest tutaj zbędna